README.md

```diff
@@ -5,88 +5,109 @@
+1. Include the [JitPack](https://jitpack.io/#klaviyo/klaviyo-android-sdk) repository in your project's build file
+   <details>
+      <summary>Kotlin DSL</summary>
+
+      ```kotlin
+      // build.gradle.kts
+      allprojects {
+          repositories {
+              maven(url = "https://jitpack.io")
+          }
+      }
+      ```
+   </details>
+
+   <details open>
+      <summary>Groovy</summary>
+
+      ```groovy
+      // build.gradle
+      allprojects {
+          repositories {
+              maven { url "https://jitpack.io" }
+          }
+      }
+      ```
+   </details>
+
+2. Add the dependencies to your app's build file
+   <details>
+      <summary>Kotlin DSL</summary>
+
+      ```kotlin
+      // build.gradle.kts
+      dependencies {
+          implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0")
+          implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0")
+      }
+      ```
+   </details>
+
+   <details open>
+      <summary>Groovy</summary>
+
+      ```groovy
+       // build.gradle
+       dependencies {
+           implementation "com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0"
+           implementation "com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0"
        }
-   }
-   ```
-   ```groovy
-   // build.gradle
-   allprojects {
-       repositories {
-           maven { url "https://jitpack.io" }
-       }
-   }
-   ```
-2. Add these dependencies to your app's build file
-   ```kotlin
-   // build.gradle.kts
-   dependencies {
-       implementation("com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0")
-       implementation("com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0")
-   }
-   ```
-   ```groovy
-   // build.gradle
-   dependencies {
-       implementation "com.github.klaviyo.klaviyo-android-sdk:analytics:2.0.0"
-       implementation "com.github.klaviyo.klaviyo-android-sdk:push-fcm:2.0.0"
-   }
-   ```
-
-## Analytics SDK
-
-### Configuration
-
-The SDK must be configured with the public API key for your Klaviyo account.
+      ```
+   </details>
+
+## Initialization
+The SDK must be initialized with the public API key for your Klaviyo account.
 We require access to the `applicationContext` so the SDK can be responsive to
-changes in network conditions and persist data with `SharedPreferences`.
-You must also register the Klaviyo SDK for activity lifecycle callbacks per the example code:
+changes in network conditions and persist data via `SharedPreferences`.
+You must also register the Klaviyo SDK for activity lifecycle callbacks per the example code,
+so we can gracefully manage background processes.
 
 ```kotlin
+// Application subclass 
 import android.app.Application
 import com.klaviyo.analytics.Klaviyo
 import com.klaviyo.analytics.lifecycle.KlaviyoLifecycleMonitor
 import com.klaviyo.push.KlaviyoPushService
 
 class TestApp : Application() {
     override fun onCreate() {
-        super.onCreate()
```

Comment:
Should we still be explicit and keep this line in?

Reply:
Sure, how about I include it here in the first snippet about `onCreate` but leave it out in subsequent ones (I think we refer to `onCreate` 2-3 more times

Reply:
I think thats fine ya. I feel like it should be common sense for any seasoned Android dev to typically keep the super call in but you never know